lib/AST/LayoutConstraint.cpp

```diff
@@ -106,6 +110,24 @@ bool LayoutConstraintInfo::isNativeRefCountedObject(LayoutConstraintKind Kind) {
   return Kind == LayoutConstraintKind::NativeRefCountedObject;
 }
 
+bool LayoutConstraintInfo::isClass(LayoutConstraintKind Kind) {
+  return Kind == LayoutConstraintKind::Class;
+}
```

Comment:
Should this return true when `kind == LayoutConstraintKind::NativeClass`? It "is" a class constraint, after all.